packages/wallets/src/components/FlowField/FlowTextField.tsx

```diff
@@ -34,9 +35,14 @@ const FlowTextField = forwardRef(
         };
 
         useEffect(() => {
-            if (defaultValue) {
-                setFormValues(name, defaultValue);
-            }
+            const setFormValuesAndTouch = async () => {
+                if (defaultValue) {
+                    await setFormValues(name, defaultValue, true);
+                    setFieldTouched(name, true, true);
+                }
+            };
+
+            setFormValuesAndTouch();
```

Comment:
Set field as touched everytime form value is set to ensure default value is properly set into it. This is to ensure validation can still happen if field hasn't been touched and default value from getSettings already exists.

Reference: https://github.com/jaredpalmer/formik/issues/2059
cc: @adrienne-deriv @shahzaib-deriv